festipals/src/components/Pals.js

```diff
@@ -13,13 +16,13 @@ export default class Pals extends Component {
             <span className="fa fa-user fa-3x"></span>
           </div>
           <div className="col-xs-9">
-            <h3> Friend&rsquo;s Name</h3>
+            <h3> {pals.first_name} {pal.last_name}</h3>
```

Comment:
Watch out: here it says pal**s**.first_name but pal.last_name. It probably should be something like `{this.props.firstName}`, depending on how we pass the props from the PalsPage